src/browser/api/window.js

```diff
@@ -799,8 +799,8 @@ Window.create = function(id, opts) {
         _window: browserWindow
     };
 
-    const { data } = coreState.getStartManifest();
-    const { plugin: plugins } = (data || {});
+    const { manifest } = coreState.getManifest(identity);
+    const { plugin: plugins } = manifest || {};
     winObj.pluginState = JSON.parse(JSON.stringify(plugins || []));
```

Comment:
:neckbeard: (just a suggestion) update so it looks the same as in `plugins.ts`
```js
const { plugin: plugins = [] } = manifest || {};
winObj.pluginState = JSON.parse(JSON.stringify(plugins));
```

Reply:
I totally agree with you, I tried that initially, but the linter or the transpiler did not like defaults in the de-structuring (don't remember which one)...